R/fotmob_leagues.R

```diff
@@ -0,0 +1,246 @@
+
+.load_fotmob_leagues <- function() {
+  read.csv("https://raw.githubusercontent.com/JaseZiv/worldfootballR_data/3c6ff713a08a0ef5f9355b8eba791a899fe68189/raw-data/fotmob-leagues/all_leagues.csv", stringsAsFactors = F)
```

Comment:
this URL should be changed before this PR is merged! https://github.com/JaseZiv/worldfootballR_data/pull/2 should be merged first, which will change the url